### ISSUE TITLE
FNSO 2019 Call for Funding 

### ISSUE THREAD

**Issue by spapastamkou** [programminghistorian/jekyll#1659]
As I said in our #1589 January meeting, the National Fund for Open Science in France recently released a call for funding; technical details (see last 2 attached docs) have been been available through a dedicated application that opened 2 weeks ago; the call runs up to March 31. All documentation is available in English as well. 
[Link to the call](https://www.ouvrirlascience.fr/call-for-projects-national-fund-for-open-science/) 
[FNSO_2019_CALL_FOR_PROJECTS_(english_translated_version).pdf](https://github.com/programminghistorian/jekyll/files/4141415/FNSO_2019_CALL_FOR_PROJECTS_.english_translated_version.pdf) - annotated some points
[FNSO_2019_Preparatory_template_for_the_on-line_form_(english_translated_version).pdf](https://github.com/programminghistorian/jekyll/files/4140835/FNSO_2019_Preparatory_template_for_the_on-line_form_.english_translated_version.pdf)
[FNSO_2019_Financial_request_table_(english_translated_version).xlsx](https://github.com/programminghistorian/jekyll/files/4140838/FNSO_2019_Financial_request_table_.english_translated_version.xlsx)

I believe we are ok for the category of innovative editorial contents (funding min 10k € - max 45k €)

What I have done up to now: 
- read the call but not all tech details; assured we fill most of the exemplary criteria;
- assured I am eligible to coordinate despite non Lecturer position (I believe I am the only member of the FR team with a stable position, which by the way can be an issue we need to consider in the future);
- assured that a former colleague (from my previous establishment) who is specialized on building projects can help to prepare the technical parts (budget, assemble the project correctly etc) - or at least she can review; 
- have asked to an official translation agency (a partner service of the CNRS at the University of Nancy) for estimation of costs for translations of PH tutorials. These are considered specialized translations and, project or not, it would be good to know. 

There are much more things to do to build a bid correctly:-) But before anything, please @acrymble and @drjwbaker take a look and let me know what you think about the PH fitting in it so that we can work on a proposal for the PH FR. @programminghistorian/french-team

**Comment by acrymble** [comment]
@spapastamkou what do you have in mind to request funding for? What would the project be?

**Comment by spapastamkou** [comment]
I thought we'd propose to get funding for the translation of a batch of tutorials, which would allow us to focus on reviewing these translations; and would liberate us for pursuing the creation of original lessons in French (the more representative of all francophone areas possible).

**Comment by spapastamkou** [comment]
This said, any ideas welcome here.

**Comment by acrymble** [comment]
That would be great @spapastamkou, but I don't think that would be covered by the scope of the call.
<img width="881" alt="grant" src="https://user-images.githubusercontent.com/646647/73692093-8f050e00-46cb-11ea-8034-ff75e4e8a66f.png">

**Comment by acrymble** [comment]
were you thinking about a different one of the funding aims?

**Comment by spapastamkou** [comment]
Rather the: c) Innovative editorial forms. Defined as: "Innovative editorial forms concern projects that do not use traditional editorial formats. These are formats that renew the form of journals or reinvent them." It is quite large, and it seemed to me the PH fits in.

**Comment by acrymble** [comment]
One of the key judgment criteria is how far the project will push the aims of open scholarship. You might want to choose the proposed translations carefully to make sure they promote that aim. Maybe proposing an event like Bogota but in France to seek new tutorials on open science would make the proposal stronger? Just some ideas.

**Comment by spapastamkou** [comment]
Yes, absolutely. I had previously made a small selection of tutorials in this spirit but we could organize it better and expand it. And if the form of the proposal allows for it, we could maybe seek to pursue our Senegalese dream:-)

**Comment by drjwbaker** [comment]
So do we think we are a good fit? On a practical level, I've got a lot on right now, but could find time to look at an early draft.

**Comment by spapastamkou** [comment]
I could try to have one ready for let's say the 15th Feb: we can take a look then and see how solid a proposal we have and further work on it.

**Comment by drjwbaker** [comment]
Fab. And if you could produce (alongside the draft) a note on two on how **you think** the draft hits the terms of the call, that would be really useful from a Critical Friend perspective.

**Comment by spapastamkou** [comment]
Small update (a bigger should come later in the week): 
I work on [a googledoc](https://docs.google.com/document/d/1qOKdjeeqqkxsgCduOlVYfiU19UeYjZV8ASLqLI8Ou4c/edit?usp=sharing) in FR on the basis of two objectives: 1) get funding for a batch of translations; 2) get funding for a writing workshop related to OS tutorials. 

For the objective 1: I have asked and obtained translation cost estimations from the Institute of scientific and technical information of the CNRS; colaterally, they proposed that we make a partnership for the sake of this grant and will have further talks on Wednesday afternoon:-) Will report back then. This could be a good card, I think. 
For the objective 2: I have sollicited the University Cheikh Anta  Diop, Dakar, Senegal via our contact with whom we had collaborated for a former grant proposal (with @acrymble) - awaiting for his response to see if we could have them as partners for a writing workshop with in-kind participation. 

I still need to check technical details for building the budget etc, but the person I should work with is on vacation this week. 

That's all, hope something will come out. Will resume main points of the proposal in EN once I have more info from possible partners.

**Comment by spapastamkou** [comment]
All - it took a bit more time than I thought but here are the latest news. There is a first draft in the googledoc but I still have to work on the detailed budget. 
As it is, the bid proposal has:  
- 3 beneficiary parts (direct recipients of funds): a) my research unit via the regional interdisciplinary institute jointly supported by the 3 regional universities and the CNRS => funding is in more favorable conditions if via a FR public research unit; b) ProgHist Ltd.; c) the National Institut of Scientific and Technical Information - Inist (CNRS unit)
- 2 stakeholders (not direct fund, and some contribution on their part): a) the U of Cheikh Anta Diop in Dakar, Senegal; and b) Institut français de recherche en Afrique - IFRA-Ibadan (Nigeria) - this has yet to be confirmed

The 2 objectives I defined until now: 
1) have a batch of tutorials translated from professionals (Inist)
2) Organize a writing workshop in Senegal for open science oriented tutorials. 

For the objective 1: I expect cost estimations for some 50 tutorials from the Inist
For the objective 2: I expect some information from Dakar to have cost estimations for the budget I need to work on. I hope I will be able to have contribution from French scientific cooperation institutions in Africa -this is where the IFRA-Ibadan comes into play, I have obtained a contact with its director via a French historian. Will also look from the side of the Institut français au Sénégal. 

What we need to see internally for these 2 objectives:
 For the objective 1: prepare a summary of clear instructions ot translators; the guidelines are ok but they are for volunteers coming form the community. This can involve simple things such as the format file we'll provide, the format file we expect, ask clearly that translated content will be under CC BY etc. It also means we shall have to organize the reviews between us and make corrections ourselves (this changes the nature of our peer review, I think). 

For the objective 2: I would appreciate views whether a workshop in Senegal, albeit a good card to ask the bid, is the best idea, or if we should consider parallel workshop(s) in France, Quebec or other francophone country, in order to be sure we can have sufficient production of lessons. I do not have a clear opinion on this. Parallel worksops mean bigger budget and in this case we should see whether, for example, @fdlaramee can/want to organize something in Montréal, look for support of a local institution etc. Or this could perhaps be something in collaboration with Humanistica (maybe for the francophone conference of 2021??) in which case it would gather easily all members of the team and relevant participants. Any ideas, @fdlaramee @grandjeanmartin @mariechristineb ? I'll try to write you as well for these matters. 

Finally, I want to ask if there is a well delimited task in relation with the implementation of the S plan and PH compliance we could ask funding for; or maybe the FR copy-editing chart? Other? This type of funding could go directly to ProgHist Ltd (one of the directly beneficiary parts). @acrymble ?

**Comment by acrymble** [comment]
Hi Sofia,

Thanks for sending this. I don't see why you can't expect your translators
to respond to corrections. That should be built into their contract (one
round of revision at the direction of the editor).
It might be worth translating a Spanish lesson too, since there are a few
of those now. This makes it look more global.
Make sure the African partners are being fairly compensated and treated
like partners. It seems notable to me that your "partners" are all European
and your "other parties" are African.
You know your funders better than me. I don't see why a workshop in Senegal
wouldn't be well regarded. Doing two workshops is a lot of work, so don't
over commit.
I'd suggest talking to James about the budget. He can help to make sure all
of our costs are met. We could provide a letter of support if it would be
helpful.
In terms of Plan S, we don't even know yet what the final outcome will be,
so it's difficult to ask for support at this stage. This could be bundled
in with an Administrative Support fee to go to ProgHist Ltd, which we could
then use for Plan S or something similar. I don't know that it would be
coherent to add that in with this bid, as it will feel like too many
different projects rather than one thing that comes together.

**Comment by fdlaramee** [comment]
Thanks Sofia. Do we know how many original lessons were produced (or are in the process of being produced) as a result of the Colombia workshop? It seems to me that we'd be lucky to get as much traction in Senegal, unless Humanistica happens to be held there in 2021. 

As for organizing two workshops, I know I wouldn't have the bandwidth to do it and I suspect it would overtax our small team, so maybe we should aim for a single workshop, at Humanistica wherever it happens to take place?

**Comment by spapastamkou** [comment]
Thank you @fdlaramee. 
- I think there are some 2 original lessons in ES and one more to come or something like that. Maybe @rivaquiroga can give us more exact info.  
- We stick to 1 workshop then. I think it really is a good idea to have it in Dakar.
@acrymble Thanks for the remarks, I note all. If part of the badget goes directly to Dakar, this means that they shall have to dedicate someone for managing this money and do logistics work - which I do not know if it is possible for them. But I will ask, and if it is ok, that fits me fine.

**Comment by acrymble** [comment]
@spapastamkou keep in mind that given the concerns about the virus, a lot of events are being cancelled right now, on short notice. You may want to factor that into your bid.

**Comment by spapastamkou** [comment]
I received the cost estimations for the translation of 50 tutorials: it is estimated to some 37 000 € on the basis of a 12-month contract for someone to work on them full time. Given that max funding to ask is 45 000 €, this means that the bid could not cover much else besides this. I am a bit mitigated on this, perhaps it would be better to ask for less lessons to translate. 

Otherwise, the UCAD partners in Senegal prepare a bid proposal themselves as well (independently of the PH); which means we'd better have them as non directly beneficiary partner bcz they do not have the human ressources necessary to handle budget from 2 bids right now; and this in my sense could be prejudicial for both proposals.

**Comment by acrymble** [comment]
It's difficult to offer advice because of my limited French. If they are going to hire someone specifically to do your work, why wouldn't you just hire them yourself through ProgHist Ltd? I don't see the advantage of working with an external company.

If I was reviewing the bid, I think I'd also question how the two things you're trying to achieve are related.

**Comment by spapastamkou** [comment]
@acrymble The terms of funding are more favorable for scientific establishments rather than enterprises (this is what the ProgHist Ltd will be considered). The Inist is a research unit of the CNRS.  

The batch of translations and the workshop are presented as two objectives for reinforcing a francophone open educational resource - in my knowledge, there are no other currently - first by giving body to the FR edition through a massive translation of tutorials and in parallel through encouraging the production of original tutorials from the broader francophone space. 
We can organize a skype meeting for those interested to take a closer look, perhaps?

**Comment by acrymble** [comment]
Sorry @spapastamkou I may just be misunderstanding. Who would be the manager of this person, and who would hire them? Is that you? If not, it strikes me as unusual for a project like this (but I recognize France may be different).

I think in future we'll have to start thinking about a memorandum of understanding between ProgHist and universities we work with. We've got a bit of a funny model of outsourcing to universities, but we have to make sure we're still in charge when it comes to our publications! That's a separate issue though.

**Comment by spapastamkou** [comment]
@acrymble This specific call can be built around
- 1 basic coordinator person and beneficiary establishment (receiving direct funding for a mission) 
- Beneficiary partners (receiving direct funding for a mission)
- Partners (not touching funding but bringing their own financial or in kind contribution)

I had first put ProgHist Ltd as basic coordinator but the funding is less interesting in this case (companies can be funded at most 45%; public scientific entities at 100%). Which is why I put my unit as coordinator (it had to be present anyway as I am staff) and did so via the Regional interdisciplinary institute which has capacity to manage budget and is part of a national research  infrastructure for Humanities - a network very much favored by the CNRS. The main task would be handling the budget for the workshop. 

ProgHist Ltd is now defined as beneficiary partner which is why I asked if we have well delimited task we can put forward and ask funding for. Otherwise I could split the budget for the workshop and ProgHist Ltd could manage the part that is relevant for its members' participation to it. 

The Inist (Institut national de l'information scientifique et technique), is a research and service unit of the CNRS which among other things provides professional services of translation to academics via their research unit. I contacted them at first to have cost estimations for translations (I had a hard time finding others anyway); it turned out they were interested to have a partnership. In the draft - which is a work under progress - they are presented as beneficiary partner as well bcz it is technically easier if they get direct funding for services they will provide (translation) - instead of having the coordinator establishment handling funding for these services and transmitting it to them. Their prices are VAT-free for scientific establishments; they are not for other customers. In any case, as it is a batch of translations we have discussed, rather than price them individually, they proposed to hire someone to work on the batch full-time. Following this scheme, the manager would be the director of this unit, but essentially the person in charge of translations with whom I exchanged, and following a well-defined memorandum previously agreed between us. 

I had the idea to build the bid around decentralized tasks according to what any of the partners can do best without overcharging any of them, us included. If we ever have a big funding, I think this questions also our capacity, actually.

**Comment by acrymble** [comment]
Thanks @spapastamkou I think I understand better now.

I would strongly encourage you to form a relationship with this partner whereby you are the manager of the translator. Otherwise I'm concerned that you're giving up editorial control. Is the rest of the @programminghistorian/french-team team happy with the proposal?

**Comment by drjwbaker** [comment]
On..

> ProgHist Ltd is now defined as beneficiary partner

.. what does this specifically mean in terms of:

- € to be paid to ProgHist Ltd
- € to be spent by ProgHist Ltd
- services delivered by ProgHist Ltd

**Comment by fdlaramee** [comment]
I concur with @acrymble -- We would benefit from a structure that allows you, @spapastamkou, to have direct oversight of the translator. I have worked as a translator on projects where the client wasn't directly involved from the start and they did not always go very well. Compounding the issue is that we have rather unusual content that includes source code that needs to be translated (and tested to make sure that it still works), etc. There will certainly be some adjustments to make early on.

I like the proposal and have sent Sofia some ideas (that may or may not be feasible given the parameters of the competition) to help manage our workflow. I'll defer to her judgment in this matter.

**Comment by spapastamkou** [comment]
> On..
> 
> > ProgHist Ltd is now defined as beneficiary partner
> 
> .. what does this specifically mean in terms of:
> 
> * € to be paid to ProgHist Ltd
> * € to be spent by ProgHist Ltd
> * services delivered by ProgHist Ltd

@drjwbaker 
(definition of the beneficiary): "The Beneficiary is a legal entity receiving funding for a Project selected following the FNSO 2019 call for projects."

**€ to be paid to ProgHist Ltd:**
From what I understood from the call: ProgHist Ltd should be considered as a small and mediumsized enterprise (SME) and this means eligibility to _de minimis_ aid with maximum aid rate at full cost: 45%.  
**€ to be spent by ProgHist Ltd**
The remaining 55%
**services delivered by ProgHist Ltd**
I believe this should be realizing the task for which the aid is asked (if I understand correctly the question). 
For example, this could let the possibility to ask for max 45% of the estimated cost for realizing a French copy-editing chart; or the same percentage for a service that assures to the PH compliance with open science principles (try to cover a part of the costs for the DOIs? other objectives in relation with the S plan?)

**Comment by spapastamkou** [comment]
@acrymble @fdlaramee 
- normally the terms of collaboration between partners will be explicit - this what was proposed by the guy of the Inist. 
- the cost estimations were made on the basis of special texts to translate, because of the hybrid form they have, combining humanities and social science vocabulary with use of code. 
- define myself as manager of the translation, if I get it correctly (and if we ever get the fundong, of course!), would mean undertaking essentially the editorial work for at least 30 translations. Even divided in 3 years time, this could be a lot and would make me almost the exclusive editor of the PH FR. I think that for the sake of the proposal we should at least state 2 members, say me and the future Managing Editor(s), and even state that in case of success the team should seek a new member to be the 3rd dealing with the other 2 with these tasks. What do you think?

**Comment by acrymble** [comment]
@spapastamkou if the spending rules are similar in France to in the UK, then the 55% that ProgHist would be expected to cover would be for direct costs only. So if you're going to hire your translator through a university then that part of the grant has no financial impact on ProgHist. If we wanted to hire someone at ProgHist, the grant would  presumably only agree to cover 45% of their salary.

So what we need to know is exactly how much of that money would be coming through ProgHist, and exactly what would the 55% be in Euros. Remember, only @drjwbaker can authorise spending and this would need to be approved in advance of any submission. We can't have ProgHist bound to debts it can't afford to pay.

**Comment by spapastamkou** [comment]
@acrymble If we take the example of the Fr copy-editing cost estimation, this was £1800 if I remember correctly. In this case, if we ask for £ 810 (which is the 45%) in this grant, will it be possible to have the rest covered by ProgHist Ltd?

**Comment by acrymble** [comment]
You'll have to ask @drjwbaker when he gets back next week. He will probably want to see a full budget with all items on it (those to be covered by ProgHist and those paid in other ways). Usually you'd also add an administrative fee for ProgHist to cover the costs of doing whatever work is required. The scale of the fee depends on how much work is involved. This is something you'll have to define.

If you want to have a call to discuss this I am happy to.

**Comment by spapastamkou** [comment]
@acrymble sure, what are your availabilities? I'll do my best to get information in the meantime to see if and how administrative fees can be introduced in the budget.

**Comment by drjwbaker** [comment]
Okay. I'm back. In short, ProgHist Ltd cannot afford any uncosted activity. That is, we the company is being asked to pay for 55% of something, we can't afford that. Or an I misunderstanding the FNSO funding model?

**Comment by spapastamkou** [comment]
Latest news: due to the extraordinary conditions, the deadline is now 30 April. 

@drjwbaker Normally this is it. However, as the PH is non-profit, there might be a way to avoid this contribution of 55%, as the person who advised on the bid building told me. We were supposed to talk last Tuesday, but it was finally impossible. I'll try to find out more next week (I want to hope we'll get used to our new rythme) and will let you know further.

**Comment by drjwbaker** [comment]
@spapastamkou Okay. I appreciate that there are language barriers here (the barrier being that my French is rubbish), but if we can get assurances that ProgHist Ltd won't be expected to spend money it doesn't have, that would be amazing.

**Comment by spapastamkou** [comment]
I resumed the work on writing the proposal after some time of interruption. Considering the situation created by the Covid19 pandemic and the impact it has on the organisation of physical scientific events, @acrymble had proposed during a discussion we had that rather than proposing a workshop in Senegal, we hire someone instead to produce (or have produced,?) some lessons specifically considering issues of particular interest for West African public - it seems for example that cultural heritage preservation subjects could be particularly useful (based on an exchange I had with a Canadian scholar who founded an OA editorial platform for African journals). This makes absolutely sense and is in terms with the kind of collaboration we are interested in establishing with partners in these countries, I think. 

Thus, it would make sense to ask for funding of producing this kind of lessons. But in order to translate this in terms of cost and ask for funding, I have (at least) two problems: 1) I find it absolutely impossible to estimate the worth of authoring a lesson (it was what I had in mind first) 2) Even admitting we have a solution for this, then it would create a different pattern of collaboration, as all of our lessons are written from volunteers for free. I do not know if this is desirable. 
So I wonder whether we can consider the following and I welcome views on these: 1) Agree that we (we = at least one of @programminghistorian/french-team and one of @programminghistorian/global-team) work on close cooperation with Florence Piron (of U Laval, Canada, the Canadian scholar I mentionned above), Abdou beukeu Sow of U Cheikh Anta Diop, eventually one more member from the Université virtuelle du Sénégal to agree on areas of interest for authoring lessons (they all have a close knowledge of local/regional needs and not only in Senegal but also in the other francophone countries). Then we look for authors who could write these lessons, either from the directly interested countries, or from other countries where the same areas of interest have found solutions, or two authors in collaboration, one local/regional one external (if pertinent). Always on a volunteer basis. And the PH could take in charge editing/copy-editing costs, and for these costs we could ask funding. How do you feel for something like that, does it make sense to you?

**Comment by acrymble** [comment]
If you are looking to cost the writing of lessons, writing is usually paid by the word or by the assignment. There are writer's associations around the world that can tell you the current going rate for different types of writing.

Not all of our authors have worked for free. Many do it as part of their paid job. And the Bogota participants, one might suggest, were paid.

**Comment by acrymble** [comment]
@spapastamkou are there a couple of specific lessons you could suggest, and I can try to help you cost out what a fair rate might be? The lessons are so different in length and complexity.

**Comment by spapastamkou** [comment]
@acrymble Do we have an idea of the actual cost of PH tutorials that have been written? Or, if you and other authors (@drjwbaker ? @fdlaramee?) can help me by estimating how many hours writing a tutorial took you, I could draw an estimate according to a FR researcher's salary so that it speaks to a FR founding institution. What do you think?

**Comment by spapastamkou** [comment]
@acrymble Something like [this lesson on Omeka](https://programminghistorian.org/en/lessons/creating-an-omeka-exhibit) or the lesson we look for [on choosing a metadata schema](https://programminghistorian.org/en/lesson-requests) (it should be longer, I guess), or like James' on preserving research data.

**Comment by acrymble** [comment]
The last lesson I wrote (a very long one) https://programminghistorian.org/en/lessons/gravity-model, was about 3 weeks work if I remember correctly. It required a lot of research and time writing. Some other lessons are much shorter. So I'd say 3 weeks is a safe upper estimate.

**Comment by fdlaramee** [comment]
@spapastamkou I did not keep detailed records but from my 2017-18 calendars I can estimate that writing and revising the stylometry tutorial took about 35-45 hours. It is hard to generalize from my experience because 1. the estimate does not take into account the time I had spent coding and writing the seminar project on which the lesson is based, two years earlier; 2. the lesson is a big one (some 8,600 words); and 3. I have been a professional writer for a very long time. 

I'd go with @acrymble's estimate, as long as the authors don't have to create too much new code for the lesson, in which case all bets are off.

**Comment by drjwbaker** [comment]
Always over-estimate how long it takes to write :) But yeah, given the process we have, 3 weeks spread over 3 months (so roughly 1.25/1.5 days per week, or 0.25/0.3 full time equivalent, over 3 months) seems a fair estimate.

**Comment by spapastamkou** [comment]
Thanks, this helps greatly, @acrymble, @fdlaramee and @drjwbaker !

**Comment by spapastamkou** [comment]
While working on the part of the grant entitled "Exemplary criteria" - we fulfill most of them - I have questions coming about two points essentially: accessibility issues, scientific integrity. I will open separate tickets to make the discussion easier and bcz I think they are of more general interest fo the PH. 

I attach again here the list of the criteria in case someone wants to have a look. 
[exemplarity-criteria-editorial-content.pdf](https://github.com/programminghistorian/jekyll/files/4496925/exemplarity-criteria-editorial-content.pdf)

**Comment by spapastamkou** [comment]
Note the deadline was (again) modified yesterday: new date is June 1st.

**Comment by acrymble** [comment]
Are you waiting on anything from any of us? I don't want you being held up (especially if I can help). I note you have opened 2 tickets about policies. Anything else?

**Comment by spapastamkou** [comment]
We are on the budget part with the person who helps to build the bid. I may have one things or two to discuss with the global team (esp. the African part) and you, but it won't be until some days and it woud be to finalize. I'll let you know by e-mail most probably. Thanks!

**Comment by spapastamkou** [comment]
I post here a document with a budget summary for two options. Both ask for funding for translations and (option 1) a workshop in Dakar, Senegal in 2021 OR (option 2) hiring and paying authors for 5 lessons on areas that would be of special interest for African countries.

Option 1 was the original idea, option 2 came up after the Covid19 crisis and its impact on international events. I assembled both of them however to be sure I had at least one eligible proposal, as the bid is eligible if services provided by 3rd parties (such as translation and authorship) do not exceed 50 % of the total demand. 

I contacted directly by e-mail Adam and Jessica but all views are welcome, if any. Option 1 could be risky bcz of the workshop in the post-Covid19 context, option 2 bcz the call favours an economic model to emerge from the demand (is authors remuneration such a model?)

Technically, both proposals have been under control and budgets are eligible. 
   
[fnso-options.txt](https://github.com/programminghistorian/jekyll/files/4629715/fnso-options.txt)

**Comment by drjwbaker** [comment]
Agree anything with travel is too risk to plan right now.

> 2) Lessons on demand (hire and pay authors for specific lessons)

Can this be tied in with/via contacts in Senegal? As in, fostering contributions towards FR-first articles from the Global South is still important to us in a post-covid world. That said, I appreciate that potential partners in Francophone Africa may have other priorities, and their priorities should drive our outreach work.

**Comment by JMParr** [comment]
I'm inclined to agree with you and James about travel. And a number of
African countries understandably don't want visitors right now anyway. We
also don't know when things will be safe for international travel again..
Option 2 is probably going to be our best bet, as long as we can work out
the logistics.

**Comment by spapastamkou** [comment]
The bid was submitted (with option 2). Thank you all for the contributions: I did not answer to all comments but I took them into consideration either for the bid or for organizing our future actions. I am happy to close this ticket:-)
